src/improvements/tasks/eth/000-opcm-upgrade-v200/VALIDATION.md

```diff
@@ -0,0 +1,869 @@
+TODO:
+- Add tenderly only sections
+- Add links to Who addresses?
```

Comment:
Yea, so as we talked about on our call. I've been adding links for every address I can by default. If I can't find a link I won't add it but the 'Details' sections will give more information.